src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -68,7 +68,8 @@
                             <h2 class="ez-fieldgroup__name">{{ 'content.view.subitems'|trans|desc('Sub-items') }}</h2>
                             <div class="ez-sil" 
                                 data-location="{{ location.id }}" 
-                                data-items="{{ subitems_module.items }}" 
+                                data-mfu-can-create="{{ content_create_permissions_for_mfu|json_encode() }}"
```

Comment:
Does `data-mfu-can-create` store `contentCreatePermissionsConfig`, which is an object? If so, I would rename this data-attribute to e.g. `data-mfu-create-permissions-config`, which for me is more precise.

Reply:
Yes, it does